crates/pathfinder/examples/fix_missing_classes.rs

```diff
@@ -0,0 +1,110 @@
+use anyhow::Context;
+use pathfinder_lib::{
+    core::{Chain, ClassHash, StarknetBlockNumber},
+    sequencer::reply::transaction::Type as TransactionType,
+    sequencer::{Client, ClientApi},
+    state::CompressedContract,
+    storage::{
+        ContractCodeTable, StarknetBlocksBlockId, StarknetBlocksTable, StarknetTransactionsTable,
+        Storage,
+    },
+};
+
+/// Add missing declared classes.
+///
+/// Iterates over all transactions in the database and checks if we have the classes downloaded
+/// for all declare transactions, downloading missing classes.
+///
+/// Usage:
+/// `cargo run --release -p pathfinder --example fix_missing_classes ./mainnet.sqlite`
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    let chain_name = std::env::args().nth(1).unwrap();
```

Comment:
There is actually functionality in `pathfinder.rs` which detects the chain from the database based on the genesis block hash.

Can maybe use just that? And move the function into `impl Storage`.. maybe?